### ISSUE TITLE
Tag a stable release for melpa-stable?

### ISSUE THREAD

**Issue by stsquad** [sjbach/lusty-emacs#23]
It's been awhile and I miss having lusty-explorer on my non-primary machines as they only use melpa-stable repos. Any chance we could tag a release so melpa-stable can start shipping (essentially the same) version as melpa?

**Comment by sjbach** [comment]
Good idea.  Done.  Let me know if it doesn't work or if there's anything else I need to do - I just guessed my way through.
